net/dnscrypt-proxy2/files/dnscrypt-proxy.init

```diff
@@ -7,6 +7,15 @@ START=18
 # stops before networking stops
 STOP=89
 
+[ -n "${IPKG_INSTROOT}" ] && return 0
+
+if type extra_command 1>/dev/null 2>&1; then
+        extra_command 'version' 'Show version information'
+else
+# shellcheck disable=SC2034
+        EXTRA_COMMANDS='version'
+fi
```

Comment:
Why do we need this? It is not clear for me, when dnscrypt-proxy2 provides this version.
See: https://github.com/DNSCrypt/dnscrypt-proxy/blob/4511c182fee62edd9ed17ac9906ec1d458e009d3/dnscrypt-proxy/main.go#L49